spec/models/invite_spec.rb

```diff
@@ -404,4 +404,28 @@
       expect(invite.invalidated_at).to be_nil
     end
   end
+
+  describe '#warnings' do
+    fab!(:invite) { Fabricate(:invite) }
+    fab!(:group) { Fabricate(:group) }
+    fab!(:secured_category) do
+      secured_category = Fabricate(:category)
+      secured_category.permissions = { group.name => :full }
+      secured_category.save!
+      secured_category
+    end
+
+    it 'does not return any warnings for simple invites' do
+      guardian = Fabricate(:admin).guardian
+      expect(invite.warnings(guardian)).to contain_exactly()
```

Comment:
```suggestion
      expect(invite.warnings(guardian)).to eq([])
```

Reply:
I think `be_blank` is an even better change.